R/stao/CFEC_BIT_submission_199.R

```diff
@@ -0,0 +1,136 @@
+    stringsAsFactors = FALSE)
+attributeList1 <- set_attributes(attributes1)
+
+# generate physical
+physical1 <- pid_to_eml_physical(mn, pkg$data)
+
+# generate dataTable
+dataTable1 <- new('dataTable',
+                  entityName = 'BIT.csv',
+                  entityDescription = 'Slightly modified csv file of CFEC Basic Information Table (BIT) including number of permits, total pounds landed, total estimated gross earnings, and average estimated gross earnings per permit for each permit fishery by year, from 1975 to 2016',
+                  physical = c(physical1[[1]]),
+                  attributeList = attributeList1)
+
+# add dataTable to EML
+eml@dataset@dataTable <- c(dataTable1)
+
+# change geographic coverage
+geocov1 <- new("geographicCoverage", geographicDescription = "The geographic region includes all commercial fishery management areas in Alaska as well as all areas of residence of permit holders.",
+               boundingCoordinates = new("boundingCoordinates", 
+                                         northBoundingCoordinate = new("northBoundingCoordinate", 72),
+                                         eastBoundingCoordinate = new("eastBoundingCoordinate", -129),
+                                         southBoundingCoordinate = new("southBoundingCoordinate", 51),
+                                         westBoundingCoordinate =  new("westBoundingCoordinate", -179)))
+eml@dataset@coverage@geographicCoverage <- c(geocov1)
+
+# change abstract
+eml@dataset@abstract <- new("abstract", .Data = "The Commercial Fisheries Entry Commission (CFEC) is an independent, autonomous agency of the State of Alaska which regulates entry into Alaska's commercial fisheries. The CFEC is committed to promoting conservation and sustained-yield management of Alaska's unique fishery resources and boosting economic stability among fishermen and their dependents. The CFEC is responsible for leasing all commercial fishing permits including permits for limited-entry fisheries. Limited-entry fisheries include all state salmon fisheries, most herring fisheries, and various other fisheries. The number of permits, total pounds landed, total estimated gross earnings, and average estimated gross earnings per permit are provided for each permit fishery by year, from 1975-2016. Information is subtotaled by resident type. Estimated permit values at year-end are shown for limited fisheries with a sufficient number of permit sales. The data included in this package has been modified slightly from what was provided by CFEC: currency columns were reformatted as numeric, and an empty column was removed.")
+
+# write and validate EML
+write_eml(eml, eml_path)
+eml_validate(eml_path)
```

Comment:
generally better to `eml_validate` before you `write_eml` just so you don't overwrite your file with bad eml!  So that'd be:
```
eml_validate(eml)
write_eml(eml, eml_path)
```